FlorenceBML/BMLacq681/BMLacq681.xml

```diff
@@ -373,30 +455,74 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                     </msContents>
                         <physDesc>
                             <objectDesc>
-                                <layoutDesc><layout columns="2" writtenLines="25"></layout></layoutDesc><!-- copied from part 2 - or different? -->
+                                <layoutDesc>
+                                    <layout columns="2" writtenLines="21" corresp="p3">
```

Comment:
if you want to have `xml:id `  textarea5 (not necessary!) you should assign it to the `layout `element here